task-manager-backend/src/user/usecases/signup-user/signup-user.js

```diff
@@ -0,0 +1,26 @@
+const userMongo = require("../../mongo/user-mongo");
+const ApiError = require("../../../utils/api-error");
+const { sendMailUsecase } = require("../../../mail/usecases");
+const removeExcludedFields = require("../../../utils/remove-excluded-fields");
+const {
+    CREATE_FAILED,
+    USER_EXCLUDED_FIELDS,
+} = require("../../../commons/constants");
+
+exports.execute = async (userData) => {
```

Comment:
follow the breaking format and the error comment as well

Reply:
okay